paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -75,12 +78,14 @@ internal class DefaultLinkAccountManager @Inject constructor(
             }
 
     override suspend fun lookupConsumer(
-        email: String,
+        email: String?,
```

Comment:
wonder if we want to have different entry points in `LinkAccountManager` now, even if the underlying endpoint is the same, to avoid confusion:

- `lookupByEmail()`
- `lookupByLinkAuthIntent()`

That way we'd avoid nullability on these parameters that are mutually exclusive right?